app/src/main/resources/application-dev.yml

```diff
@@ -28,3 +28,11 @@ micronaut:
 ---
 app:
   service-id: ${LIBRE311_SERVICE_ID}
+
+datasources:
+  default:
+    url: jdbc:mysql://localhost:3306/libre311?allowPublicKeyRetrieval=true&useSSL=false
```

Comment:
Thoughts on having the port the same as Unity Auth's (13306)? That way, in our local dev environments, we can run one MySql server and have two separate database. In the same vein, we can rename the files to be the same and change Auth service's database name from `test` to something more descriptive.

Reply:
Yeah I like that, I'll give it a try

Reply:
I'm going to merge this and try that out on the following PR (the one about local dev environment enhancements)